lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/util/JsonUtils.java

```diff
@@ -163,7 +164,9 @@ public static JsonFactoryBuilder jsonFactoryBuilder()
                         .maxStringLength(Integer.MAX_VALUE)
                         .maxNestingDepth(Integer.MAX_VALUE)
                         .maxNumberLength(Integer.MAX_VALUE)
-                        .build());
+                        .build())
+                // https://github.com/FasterXML/jackson-core/issues/1256
```

Comment:
How about moving this to the top? I think it would read better

Reply:
Sure!